src/fsharp/TcGlobals.fs

```diff
@@ -1192,6 +1192,7 @@ type public TcGlobals(compilingFslib: bool, ilg:ILGlobals, fslibCcu: CcuThunk, d
   member val attrib_CallerLineNumberAttribute = findSysAttrib "System.Runtime.CompilerServices.CallerLineNumberAttribute"
   member val attrib_CallerFilePathAttribute = findSysAttrib "System.Runtime.CompilerServices.CallerFilePathAttribute"
   member val attrib_CallerMemberNameAttribute = findSysAttrib "System.Runtime.CompilerServices.CallerMemberNameAttribute"
+  member val attrib_SkipLocalsInitAttribute  = findSysAttrib "System.Runtime.CompilerServices.SkipLocalsInitAttribute"
```

Comment:
I'm not quite sure if this shouldn't be `tryFindSysAttrib`. Attributes above use both find and try find and I can't tell what the deciding factor is.

Reply:
This is fine. You can check to see if you can deref it. If you can't, then it couldn't find it.